SPECS/cri-o/cri-o.spec

```diff
@@ -1788,4 +1792,4 @@ mkdir -p /opt/cni/bin
 - Add cri-o package: CRI-O is meant to provide an integration path between OCI
   conformant runtimes and the kubelet. Specifically, it implements the Kubelet
   Container Runtime Interface (CRI) using OCI conformant runtimes. The scope of
-  CRI-O is tied to the scope of the CRI.
```

Comment:
Was this change intentional?

Reply:
Good catch, I completely missed that in my review. I don't think we should be modifying someone else's changelog entries. I'll update the PR in Bala's name.